### DISCUSSION TITLE
Time not right but canot start raspi-config

### DISCUSSION THREAD

**Discussion by biancapascal** [mkevenaar/FarmPi#26] [Q&A] [answered]
My system time of my pi is not right after installing farmpi.
How can i enter raspi-config in shell? its not working with sudo -i or sudo

**Comment by mkevenaar** [comment, marked as answer]
The OS is not the official Raspberry Pi OS, due to lack of support of MongoDB on that OS

It's documented in the documentation: https://farmpi.kevenaar.name/guides/setup.html#additional-changes
